src/Microsoft.VisualBasic/tests/DateAndTimeTests.cs

```diff
@@ -0,0 +1,32 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Xunit;
+
+namespace Microsoft.VisualBasic.Tests
+{
+    public class DateAndTimeTests
+    {
+        [Fact]
+        public void Now_ReturnsDateTimeNow()
+        {
+            var dateTimeNowBefore = DateTime.Now.ToUniversalTime();
+            var now = DateAndTime.Now.ToUniversalTime();
+            var dateTimeNowAfter = DateTime.Now.ToUniversalTime();
+
+            Assert.InRange(now, dateTimeNowBefore, dateTimeNowAfter);
+        }
+
+        [Fact]
+        public void Today_ReturnsDateTimeToday()
+        {
+            var dateTimeTodayBefore = DateTime.Today.ToUniversalTime();
+            var today = DateAndTime.Today.ToUniversalTime();
+            var dateTimeTodayAfter = DateTime.Today.ToUniversalTime();
+
+            Assert.InRange(today, dateTimeTodayBefore, dateTimeTodayAfter);
```

Comment:
>today [](start = 27, length = 5)

maybe also assert that time component is set to zeroes?

Reply:
Yes, we need that assertion too. Thanks!
I rebased and tried to add the assertion, but somehow VS gives me strange compile errors. System.Object not defined, …
I'm on holiday and don't have time to figure it out right now.